src/Adapter/Module/AdminModuleDataProvider.php

```diff
@@ -177,97 +187,73 @@ public function isAllowedAccess($action, $name = '')
     }
 
     /**
-     * @param AddonsCollection $addons
+     * @param ModuleCollection $modules
      * @param string|null $specific_action
      *
-     * @return AddonsCollection
+     * @return ModuleCollection
      */
-    public function generateAddonsUrls(AddonsCollection $addons, $specific_action = null)
+    public function setActionUrls(ModuleCollection $modules, ?string $specific_action = null): ModuleCollection
     {
-        foreach ($addons as $addon) {
+        foreach ($modules as $module) {
             $urls = [];
             foreach ($this->moduleActions as $action) {
+                if ($action === 'configure') {
+                    $urls[$action] = $this->router->generate('admin_module_configure_action', [
+                        'module_name' => $module->attributes->get('name'),
```

Comment:
Safer to user a getter method:
```suggestion
                        'module_name' => $module->getAttributes()->get('name'),
```